src/dsp_tools/dsp_tools.py

```diff
@@ -166,77 +166,82 @@ def program(user_args: list[str]) -> None:
+                                 password=args.password,
+                                 imgdir=args.imgdir,
+                                 sipi=args.sipi,
+                                 verbose=args.verbose,
+                                 incremental=args.incremental,
+                                 save_metrics=args.metrics)
     elif args.action == "excel2json":
-        excel2json(data_model_files=args.data_model_files,
-                   path_to_output_file=args.outfile)
+        success = excel2json(data_model_files=args.data_model_files,
+                             path_to_output_file=args.outfile)
     elif args.action == "excel2lists":
-        excel2lists(excelfolder=args.excelfolder,
-                    path_to_output_file=args.outfile,
-                    verbose=args.verbose)
+        _, success = excel2lists(excelfolder=args.excelfolder,
+                                 path_to_output_file=args.outfile,
+                                 verbose=args.verbose)
     elif args.action == "excel2resources":
-        excel2resources(excelfile=args.excelfile,
-                        path_to_output_file=args.outfile)
+        _, success = excel2resources(excelfile=args.excelfile,
+                                     path_to_output_file=args.outfile)
     elif args.action == "excel2properties":
-        excel2properties(excelfile=args.excelfile,
-                         path_to_output_file=args.outfile)
+        _, success = excel2properties(excelfile=args.excelfile,
+                                      path_to_output_file=args.outfile)
     elif args.action == "id2iri":
-        id_to_iri(xml_file=args.xmlfile,
-                  json_file=args.jsonfile,
-                  out_file=args.outfile,
-                  verbose=args.verbose)
+        success = id_to_iri(xml_file=args.xmlfile,
+                            json_file=args.jsonfile,
+                            out_file=args.outfile,
+                            verbose=args.verbose)
     elif args.action == "excel2xml":
-        excel2xml(datafile=args.datafile,
-                  shortcode=args.shortcode,
-                  default_ontology=args.default_ontology)
+        success = excel2xml(datafile=args.datafile,
+                            shortcode=args.shortcode,
+                            default_ontology=args.default_ontology)
     elif args.action == "start-stack":
-        start_stack(max_file_size=args.max_file_size,
-                    enforce_docker_system_prune=args.prune,
-                    suppress_docker_system_prune=args.no_prune)
+        success = start_stack(max_file_size=args.max_file_size,
+                              enforce_docker_system_prune=args.prune,
+                              suppress_docker_system_prune=args.no_prune)
     elif args.action == "stop-stack":
-        stop_stack()
+        success = stop_stack()
+    else:
+        success = False
 
+    if not success: 
+        exit(1)
```

Comment:
At a later point of time, I'd like to test if the CLI commands of dsp_tools.py can be executed. But for this, it's necessary to distinguish between exit code 0 and 1.